extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -33,6 +33,9 @@ const TAG = 'AMP_STORY_PANNING_MEDIA';
 /** @const {string}  */
 const DURATION_MS = 1000;
 
+/** @const {number}  */
+const DISTANCE_TO_CENTER_EDGE = 50;
```

Comment:
Distance in `px`? Percent? We should add to the name. e.g. `DISTANCE_TO_CENTER_EDGE_PERCENT`